lib/winapi/BaseTsd.h

```diff
@@ -0,0 +1,6 @@
+#ifndef __BASETSD_H__
```

Comment:
The rest of winapi uses all-lowercase filenames (like MinGW).

Reply:
Should have been fixed.